app/src/main/java/com/epfl/beatlink/ui/profile/EditProfile.kt

```diff
@@ -155,10 +167,13 @@ fun EditProfileScreen(profileViewModel: ProfileViewModel, navigationActions: Nav
                               bio = description,
                               links = profileData?.links ?: 0,
                               name = name,
-                              profilePicture = imageUri,
+                              profilePicture = "",
```

Comment:
I would suggest to put `null` instead so It clearly indicates the absence of a profile picture.
```suggestion
                              profilePicture = null,
```